src/PortingAssistantExtensionClientShared/PortingAssistantLanguageClient.cs

```diff
@@ -43,7 +43,7 @@ class VisualBasicContentDefinition
     {
         [Export]
         [Name("VisualBasicFileType")]
-        [BaseDefinition("Basic")]
+        [BaseDefinition("Code")]
```

Comment:
should this only be code for vs2019? is it still valid for 2022?

Reply:
yes, Basic is inherent from code.